### ISSUE TITLE
Create rule to allow empty if blocks if there is a comment in them

### ISSUE THREAD

**Issue by ptarjan** [jscs-dev/node-jscs#1767]
I'm fine with my developers writing this:

```
if (foo) {
  // Clearly foo is ok so do nothing
} else {
  ...
}
```

**Comment by zxqfox** [comment]
It looks like new rule value for disallowEmptyBlocks.

`allExcept: ['single-comment']` ?

**Comment by hzoo** [comment]
or just `comment`/`comments`?

**Comment by zxqfox** [comment]
Yeah. Not sure about plural, but yeah ;-)

**Comment by markelog** [comment]
@ptarjan would you like to implement such value?

**Comment by ptarjan** [comment]
Fixed by 9a9b1db9 ?

**Comment by hzoo** [comment]
Yeah looks like it was fixed thanks
